torch/csrc/jit/fuser/cuda/thnvrtc.h

```diff
@@ -30,4 +30,8 @@ extern "C" typedef struct THNVRTC {
 #undef CREATE_MEMBER
 } THNVRTC;
 
-extern "C" THNVRTC* torch_load_nvrtc();
+extern "C" 
+#ifdef _WIN32 
```

Comment:
Can you use  the macro "TORCH_API" to do these exports?

Reply:
Done.